src/lib.rs

```diff
@@ -68,6 +71,8 @@ where
     path.as_ref().is_executable()
 }
 
+
```

Comment:
Adding random whitespace isn't good practice for creating a PR. You don't want to change anything more than necessary